MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,341 @@
+        orderInfo.Add (new OrderInfo ("1006", "Hanna Moos", "Germany", "BLAUS", "Mannheim"));
+        orderInfo.Add (new OrderInfo ("1007", "Andrew Fuller", "France", "BLONP", "Strasbourg"));
+        orderInfo.Add (new OrderInfo ("1008", "Martin King", "Spain", "BOLID", "Madrid"));
+        orderInfo.Add (new OrderInfo ("1009", "Lenny Lin", "France", "BONAP", "Marsiella"));
+        orderInfo.Add (new OrderInfo ("1010", "John Carter", "Canada", "BOTTM", "Lenny Lin"));
+        orderInfo.Add (new OrderInfo ("1011", "Laura King", "UK", "AROUT", "London"));
+        orderInfo.Add (new OrderInfo ("1012", "Anne Wilson", "Germany", "BLAUS", "Mannheim"));
+        orderInfo.Add (new OrderInfo ("1013", "Martin King", "France", "BLONP", "Strasbourg"));
+        orderInfo.Add (new OrderInfo ("1014", "Gina Irene", "UK", "AROUT", "London"));
+    }
+}
+{% endhighlight %}
+
+## Binding data to the SfDataGrid
+
+To bind the data source to the SfDataGrid, set the `SfDataGrid.ItemsSource` property as follows. You can bind the data source of the SfDataGrid either from XAML or in code. 
+
+The following code example binds the collection created in previous step to `SfDataGrid.ItemsSource` property.
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:GettingStarted;assembly=GettingStarted"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid">     
+
+    <ContentPage.BindingContext>
+        <local:OrderInfoRepository x:Name="viewModel" />
+    </ContentPage.BindingContext>
+
+    <ContentPage.Content>
+        <syncfusion:SfDataGrid x:Name="dataGrid"
+                               ItemsSource="{Binding OrderInfoCollection}">
+        </syncfusion:SfDataGrid>
+    </ContentPage.Content>
+</ContentPage> 
+{% endhighlight %}
+{% highlight c# %}
+OrderInfoRepository viewModel = new OrderInfoRepository ();
+dataGrid.ItemsSource = viewModel.OrderInfoCollection; 
+{% endhighlight %}
+{% endtabs %}
+
+Run the application to render the following output.
+
+![.Net MAUI DataGrid](Images\getting-started\basic-datagrid.jpg)
+
+## Defining columns
+
+By default, the SfDataGrid automatically creates columns for all the properties in the data source. The type of the column generated depends on the type of data in the column. When the columns are auto-generated, handle the `SfDataGrid.AutoGeneratingColumnMode` mode to customize or cancel the columns before they are added to the columns collection in the SfDataGrid.
+ 
+The columns can be manually defined by setting the`SfDataGrid.AutoGenerateColumnsMode` property to None and by adding the `GridColumn` objects to the `SfDataGrid.Columns` collection. It can be done from both XAML and code. The following code example illustrates this:
```

Comment:
The columns can be manually defined by setting the`SfDataGrid.AutoGenerateColumnsMode` property to 'None' and by adding the `GridColumn` objects to the `SfDataGrid.Columns` collection. This can be done from both XAML and code. The following code example illustrates this: